docs/03-extensions/adobe-b2b/index.mdx

```diff
@@ -139,3 +138,44 @@ documents will no longer be available if this feature is disabled
   [example override](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/main/packages/theme-chocolatine/theme/modules/ProductView/Synthesis/AddProductToCart.jsx)
 
 :::
+
+## Known issues
+
+<!--------------------DETAILS-------------------->
+
+```mdx-code-block
+<details>
+  <summary className="flex justify-between">Bundle products can't be used in requisition lists</summary>
```

Comment:
**nitpick:** the issue with details is that it's not directly scannable and linkable

**suggestion:** rework this part to:
- use a heading instead `### Bundle products can't be used in requisition lists`
- add a TLDR paragraph (see next suggestion)
- rename this title as `Reproduction details and technical insights` (or any better wording you may find :))